common/reviews/api/rush-lib.api.ts

```diff
@@ -58,7 +58,6 @@ enum BumpType {
 
 // @public
 class ChangeFile {
-  // WARNING: The type "IChangeFile" needs to be exported by the package (e.g. added to index.ts)
   // @internal
   constructor(changeFileData: IChangeFile, rushConfiguration: RushConfiguration);
```

Comment:
Why exactly do we need to expose IChangeFile? It's the raw JSON file format, used by this `@internal` constructor